lib/api/src/instance.rs

```diff
@@ -17,7 +18,7 @@ use wasmer_vm::InstanceHandle;
 /// Spec: https://webassembly.github.io/spec/core/exec/runtime.html#module-instances
 #[derive(Clone)]
 pub struct Instance {
-    handle: InstanceHandle,
+    handle: Arc<InstanceHandle>,
```

Comment:
This fixes the unsafety because `InstanceHandle` is not `Sync` and thus `Arc<InstanceHandle>` is not `Send`. This does also mean that `Instance` is now also not `Send`.